frontend/src/scenes/feature-flags/FeatureFlagInstructions.tsx

```diff
@@ -242,26 +98,61 @@ export function CodeInstructions({
     const [defaultSelectedOption] = options
     const [selectedOption, setSelectedOption] = useState(defaultSelectedOption)
     const [payloadOption, setPayloadOption] = useState(PAYLOAD_OPTIONS[0])
+    const [localEvalOption, setLocalEvalOption] = useState(LOCAL_EVALUATION_OPTIONS[0])
+    const [bootstrapOption, setBootstrapOption] = useState(BOOTSTRAPPING_OPTIONS[0])
     const [showPayloadCode, setShowPayloadCode] = useState(Object.keys(featureFlag?.filters.payloads || {}).length > 0)
+    const [showLocalEvalCode, setShowLocalEvalCode] = useState(false)
+    const [showBootstrapCode, setShowBootstrapCode] = useState(false)
 
     const selectOption = (selectedValue: string): void => {
         const option = options.find((option) => option.value === selectedValue)
-        const payloadOption = PAYLOAD_OPTIONS.find((payloadOption) => payloadOption.value === selectedValue)
 
         if (option) {
             setSelectedOption(option)
         }
-        if (payloadOption) {
-            setPayloadOption(payloadOption)
-        } else {
-            setShowPayloadCode(false)
+
+        if (showPayloadCode) {
+            const payloadOption = PAYLOAD_OPTIONS.find((payloadOption) => payloadOption.value === selectedValue)
+            if (payloadOption) {
+                setPayloadOption(payloadOption)
+            } else {
+                setShowPayloadCode(false)
+            }
+        }
+
+        if (showLocalEvalCode) {
```

Comment:
I think instead of having different snippets when local eval is enabled, it should replace the main one, with comments in code explaining what's different.

Otherwise easy to confuse